src/services/dashboards/default-dashboard/helper.ts

```diff
@@ -1,21 +1,30 @@
 import { v4 as uuidv4 } from 'uuid';
 
 import { ERROR_CASE_WIDGET_INFO } from '@/services/dashboards/default-dashboard/config';
-import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/_configs/config';
+import type { DashboardLayoutWidgetInfo, InheritOptions } from '@/services/dashboards/widgets/_configs/config';
 import { getWidgetConfig } from '@/services/dashboards/widgets/_helpers/widget-helper';
 
 export const getDashboardLayoutWidgetInfoList = (widgetList: string[]): DashboardLayoutWidgetInfo[] => widgetList.map((widgetId) => {
     try {
         const widgetConfig = getWidgetConfig(widgetId);
+        const _defaultProperties = widgetConfig.options_schema?.default_properties ?? [];
```

Comment:
suggestion: The use of _ for variable names appears to be avoided by VUE's "setting function". Why don't we make our own rules?

Reply:
First, I removed `_`. But I think it would be better to discuss it together later.